Shopkeeper/Shopkeeper.Services/ItemService.cs

```diff
@@ -4,19 +4,22 @@
 using System;
 using System.Collections.Generic;
 using System.Linq;
+using Microsoft.Extensions.Logging;
 
 namespace Shopkeeper.Services
 {
     public class ItemService : IItemService
     {
         private readonly IItemRepository itemRepository;
+        private readonly ILogger<ItemService> logger;
         private Random _random;
```

Comment:
If you expect the service to be called concurrently Random isn't thread safe. Might want to add locks around call sites

Reply:
Didn't know Random isn't thread safe, I'll add the lock